### PR TITLE
Update test.py

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -1,2 +1,5 @@
+
+
+
 def main():
-    pass
+    pass
```
Comment (sils1297) [inline test.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test.py
+++ b/test.py
@@ -2,4 +2,4 @@
 
 
 def main():
-    pass   
+    pass
```

Comment (sils1297) [inline test.py]: The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test.py
+++ b/test.py
@@ -2,4 +2,4 @@
 
 
 def main():
-    pass   
+    pass
```

File: test.py [hunk 2]
```diff
@@ -1,2 +1,5 @@
+
```
Comment (sils1297) [inline test.py]: The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/test.py
+++ b/test.py
@@ -1,4 +1,3 @@
-
 
 
 def main():
```

File: test.py (hunk 1 above)
Comment (sils) [inline test.py:5]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

_SpaceConsistencyBear, severity NORMAL, section `python`._

The issue can be fixed by applying the following patch:

``` diff
--- a/test.py
+++ b/test.py
@@ -2,4 +2,4 @@


 def main():
-    pass   
+    pass
```

Comment (sils) [inline test.py:5]: The code does not comply to PEP8.

_PEP8Bear, severity NORMAL, section `python`._

The issue can be fixed by applying the following patch:

``` diff
--- a/test.py
+++ b/test.py
@@ -2,4 +2,4 @@


 def main():
-    pass   
+    pass
```

File: test.py (hunk 2 above)
Comment (sils) [inline test.py:1]: The code does not comply to PEP8.

_PEP8Bear, severity NORMAL, section `python`._

The issue can be fixed by applying the following patch:

``` diff
--- a/test.py
+++ b/test.py
@@ -1,4 +1,3 @@
-


 def main():
```

File: test.py (hunk 1 above)
Comment (sils) [inline test.py:5]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

_SpaceConsistencyBear, severity NORMAL, section `python`._

The issue can be fixed by applying the following patch:

``` diff
--- a/test.py
+++ b/test.py
@@ -2,4 +2,4 @@


 def main():
-    pass   
+    pass
```

Comment (sils) [inline test.py:5]: The code does not comply to PEP8.

_PEP8Bear, severity NORMAL, section `python`._

The issue can be fixed by applying the following patch:

``` diff
--- a/test.py
+++ b/test.py
@@ -2,4 +2,4 @@


 def main():
-    pass   
+    pass
```

File: test.py (hunk 2 above)
Comment (sils) [inline test.py:1]: The code does not comply to PEP8.

_PEP8Bear, severity NORMAL, section `python`._

The issue can be fixed by applying the following patch:

``` diff
--- a/test.py
+++ b/test.py
@@ -1,4 +1,3 @@
-


 def main():
```

File: test.py (hunk 1 above)
Comment (sils) [inline test.py:5]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

_SpaceConsistencyBear, severity NORMAL, section `python`._

The issue can be fixed by applying the following patch:

``` diff
--- a/test.py
+++ b/test.py
@@ -2,4 +2,4 @@


 def main():
-    pass   
+    pass
```

Comment (sils) [inline test.py:5]: The code does not comply to PEP8.

_PEP8Bear, severity NORMAL, section `python`._

The issue can be fixed by applying the following patch:

``` diff
--- a/test.py
+++ b/test.py
@@ -2,4 +2,4 @@


 def main():
-    pass   
+    pass
```

File: test.py (hunk 2 above)
Comment (sils) [inline test.py:1]: The code does not comply to PEP8.

_PEP8Bear, severity NORMAL, section `python`._

The issue can be fixed by applying the following patch:

``` diff
--- a/test.py
+++ b/test.py
@@ -1,4 +1,3 @@
-


 def main():
```

File: test.py [hunk 3]
```diff
@@ -1,2 +1,7 @@
...
+
+
```
Comment (sils) [inline test.py:3]: The code does not comply to PEP8.

_PEP8Bear, severity NORMAL, section `python`._

The issue can be fixed by applying the following patch:

``` diff
--- a/test.py
+++ b/test.py
@@ -2,6 +2,4 @@


 def main():
-    pass   
-
-
+    pass
```

**Pull request closed without merging**